source/includes/harvest/_tags.md

```diff
@@ -0,0 +1,165 @@
+# Tags
+
+Tags applied to a Greenhouse model, like candidate tags
+
+## The candidate tag object
+
+```json
+{
+  "id": 230,
+  "name": "New York",
+}
+```
+
+### Noteworthy attributes
+
+| Attribute | Description |
+|-----------|-------------|
+| id | The candidate tag's unique identifier |
+| name | The name of the tag |
+
+## Candidate Tags
+
+List all of an organization's candidate tags
```

Comment:
Small nitpick: our other endpoint descriptions end with a period.  Could we do the same for these as well?